app/assets/javascripts/answers.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
I'd avoid coffeescript, we don't use it here anymore, though there are some legacy bits still.